cips/notes/cdc-15.md

```diff
@@ -25,7 +25,7 @@
 *   Celestia Node to release update supporting Celestia App v2 soon
 *   Arabica devnet upgrade planned for **August 14th**
 *   Node operators must upgrade within **1 week** timeframe
-*   Consensus node operators need to use a specific CLI flag ([see documentation](https://docs.celestia.org/nodes/hardfork-process#lemongrass-hardfork))
+*   Consensus node operators need to use a specific CLI flag ([see documentation](https://docs.celestia.org/nodes/network-upgrade-process#lemongrass-network-upgrade))
```

Comment:
@rootulp I realized that I made the link "network upgrades" on accident due to the heading. so have fixed in: https://github.com/celestiaorg/docs/pull/1714